FubarDev.FtpServer.FileSystem.DotNet/DotNetFileSystem.cs

```diff
@@ -171,9 +176,18 @@ public Task<Stream> OpenReadAsync(IUnixFileEntry fileEntry, long startPosition,
         {
             var targetEntry = (DotNetDirectoryEntry)targetDirectory;
             var fileInfo = new FileInfo(Path.Combine(targetEntry.Info.FullName, fileName));
-            using (var output = fileInfo.Create())
+            try
             {
-                await data.CopyToAsync(output, _streamBufferSize, cancellationToken);
+                using (var output = fileInfo.Create())
+                {
+                    await data.CopyToAsync(output, _streamBufferSize, cancellationToken);
+                }
+            }
+            catch (OperationCanceledException)
+            {
+                if (DeletesFilesOnUploadTimeout)
+                    fileInfo.Delete();
+                throw;
```

Comment:
Na jaké úrovni se ta znovuvyhozená výjimka pak zachytí?

Reply:
zachyti se v FTP connection, kde se resi takhle:
catch (OperationCanceledException)
                {
                    // Ignore the OperationCanceledException
                    // This is normal during disconnects
                }
zaroven ale nastane System.IO exception a ta se zaloguje

Reply:
ok